help/de/Help_ch_Account-Actions.xml

```diff
@@ -1,910 +1,1049 @@
+    Systematisierung und Gruppierung der Konten zu unterstützen. Mittels dieser Kontoarten ist es
+    möglich, einen Kontenplan (Kontenhierarchie) zu erstellen.
   </para>
 
   <sect1 id="acct-types">
-    <title>Types of &app; Accounts</title>
+    <title>Arten von &app; Konten</title>
 
-    <para>&app; supports a number of different account types. It is recommended to choose an appropriate
-      account type based on the list of account types described below.
+    <para>&app; unterstützt eine Anzahl verschiedener Kontoarten. Es wird empfohlen, dass Sie aus der
+      folgenden Liste der Kontoarten jeweils die passende Kontoart auswählen.
     </para>
 
     <warning>
-      <para>The Accounts Payable and Accounts Receivable types are used internally by &app;&rsquo;s business
-        features. Transactions involving these accounts should not be added, changed or deleted in
-        any way other than by using
-        <itemizedlist>
+      <para>Die Kontenarten Verbindlichkeiten und Forderungen werden zur Verwaltung offener Rechnungen, auch
+        <quote> offfene Posten</quote> genannt, verwendet. Buchungen, die diese Konten betreffen,
+        sollten ausschließlich durch Verwendung von
+        <itemizedlist spacing="compact">
           <listitem>
-            <para>post/unpost bill/invoice/voucher or
+            <para>einbuchen/ausbuchen von Lieferantenrechnung/Rechnung/Auslagenerstattung oder
             </para>
           </listitem>
 
           <listitem>
-            <para>process payment
+            <para>verarbeiten von Zahlung
             </para>
           </listitem>
         </itemizedlist>
+        hinzugefügt, geändert oder gelöscht werden.
       </para>
     </warning>
 
+    <tip>
+      <para>Bei der Erstellung neuer Konten sind die verfügbaren Möglichkeit für die Kontoart eingeschränkt,
+        je nachdem, welche Kontoart das übergeordnete Konto (Hauptkonto) hat.
+      </para>
+    </tip>
+
     <table frame="topbot" id="TypeAccounts">
-      <title>Types of &app; Accounts.</title>
+      <title>Arten von &app; Konten.</title>
+<!-- FIXME source: insert third column -->
+      <tgroup cols="3">
+        <colspec colname="c1"></colspec>
+
+        <colspec colname="c2"></colspec>
+
+        <colspec colname="c3"></colspec>
 
-      <tgroup cols="2">
         <thead>
           <row>
             <entry>
-              <para>Account Type
+              <para>Kontoart
               </para>
             </entry>
 
             <entry>
-              <para>Description
+              <para>Beschreibung
+              </para>
+            </entry>
+
+            <entry>
+              <para>übergeordnetes Konto (Hauptkonto)
               </para>
             </entry>
           </row>
         </thead>
 
         <tbody>
           <row>
-            <entry>
-              <para>Accounts Payable
+            <entry namest="c1">
```

Comment:
Die Spalte(n) brauchst du nur angeben, wenn du welche überspringen oder verbinden willst.
Sonst würde ich die weglassen, denn wir kommen in Teufels Küche, wenn wir später davor eine Spalte einfügen wollen.